CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# Below is a list of Flutter team members who are suggested reviewers
+# for contributions to plugins in this repository.
```

Comment:
What do you think about emphasizing that it doesn't mean the default reviewer has to be the one reviewing the PR? ("suggested reviewers" says that already, but I'd feel better if we emphasize 😄  )